components/brave_wallet_ui/page/screens/onboarding/onboarding_success/components/article_link_bubble/article_link_bubble.style.ts

```diff
@@ -6,7 +6,6 @@
 import styled from 'styled-components'
 
 // icons
-import CryptoWalletsIcon from '../../../../../../assets/svg-icons/crypto-wallets-icon.svg'
```

Comment:
Please remove that file also

Reply:
Fixed in b060a4965bc59c3e1558088113acf6b85cf310a5